build.txt

```diff
@@ -0,0 +1,38 @@
+nvm install 16
+npm install --global yarn
+
+yarn install --frozen-lockfile --no-progress
+python3.9 -m venv .venv
+source .venv/bin/activate
+pip3 install slither-analyzer
+
+# compile
+yarn compile
+
+# static analysis
+yarn lint
+yarn slither
+
+# tests
+yarn test-with-coverage
```

Comment:
Error in plugin solidity-coverage: ❌ 2 test(s) failed under coverage.

  1) GenesisPurposeEscrow
       calcWithdrawableAmount
         should consider already withdrawn:

      AssertionError: Expected "225000000000000000000000" to be equal 600000000000000000000000
      + expected - actual

       {
      -  "_hex": "0x7f0e10af47c1c7000000"
      +  "_hex": "0x2fa54641bae8aaa00000"
         "_isBigNumber": true
       }
      
      at Context.<anonymous> (test/functional/GenesisPurposeEscrow.test.ts:931:12)

  2) PFP Protocol - Withdrawals and Rewards
       Protocol
         minting, bonding, interest multi-party scenario:
     AssertionError: Expected "180089934460588472179979" to be within 1000000000000000000 of 180000000000000000000000
      at Context.<anonymous> (test/integration/WithdrawalsAndRewards.test.ts:421:15)